src/main/java/com/epam/reportportal/jobs/clean/CleanMaterializedViewJob.java

```diff
@@ -51,6 +51,7 @@ public CleanMaterializedViewJob(JdbcTemplate jdbcTemplate, @Value("${rp.environm
 		this.namedParameterJdbcTemplate = namedParameterJdbcTemplate;
 	}
 
+  @Override
```

Comment:
in idea all looks good
![image](https://github.com/reportportal/service-jobs/assets/109206864/dec0561f-bd87-40d4-b06d-ad708695ab6e)